RELEASING.md

```diff
@@ -259,36 +286,35 @@ as described in the section above.
    make release-publish
    ```
 
-1. Merge the PR. This will cause candidate.docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct before proceeding to the next step.
+1. Merge the PR. This will cause docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct.
 
-1. Checkout the master branch
+1. Remove branch deploy for the candidate branch in docs.projectcalico.org site settings.
 
-1. In [netlify.toml](netlify.toml):
-
-   1. Set the `CANDIDATE_RELEASE` environment variable back to an empty string.
+## Adding the previous release to docs.projectcalico.org/archive
 
-   1. Update the `CURRENT_RELEASE` environment variable.
+1. Enable branch deployments for previous release branch on netlify by adding branch name to `Deploy contexts` in site settings.
```

Comment:
Yeah we wouldn't have to re-enable the branch builds if we left them on in the last section